app/scripts/models/auth_brokers/first-run.js

```diff
@@ -23,6 +24,11 @@ define([
       VERIFICATION_COMPLETE: 'verification_complete'
     },
 
+    defaultCapabilities: _.extend({}, proto.defaultCapabilities, {
+      chooseWhatToSyncCheckbox: true,
+      chooseWhatToSyncWebV1: false
+    }),
```

Comment:
From @shane-tomlinson feedback, we disabled CWTS in first run. 

Also this branch has the old version once we are ready to enable CWTS in first run https://github.com/mozilla/fxa-content-server/tree/cwts-firstrun-future